### PR TITLE
Make nix-install.sh script point at a binary

### PR DESCRIPTION
**Pull request by Hoverbear**
Make the `nix-install.sh` point at `install.determinate.systems` for now so we can test it internally.

### CODE REVIEW

**Comment by Hoverbear** [comment]
@grahamc also noted some `RUSTUP` instances remain which we should probably rename.

File: nix-install.sh [hunk 1]
```diff
@@ -106,9 +79,9 @@ main() {
     done
 
     if $_ansi_escapes_are_valid; then
-        printf "\33[1minfo:\33[0m downloading installer\n" 1>&2
+        printf "\33[1minfo:\33[0m downloading installer \33[4m$_url\33[0m\n" 1>&2
     else
-        printf '%s\n' 'info: downloading installer' 1>&2
+        printf '%s\n' 'info: downloading installer $_url' 1>&2
```
Comment (grahamc) [inline nix-install.sh:84]:
```suggestion
        printf 'info: downloading installer (%s)\n' "$_url" 1>&2
```

$_url won't interpolate in single quotes, and we should use printf for the purpose anyway :)

File: nix-install.sh [hunk 2]
```diff
@@ -106,9 +79,9 @@ main() {
     done
 
     if $_ansi_escapes_are_valid; then
-        printf "\33[1minfo:\33[0m downloading installer\n" 1>&2
+        printf "\33[1minfo:\33[0m downloading installer \33[4m$_url\33[0m\n" 1>&2
```
Comment (grahamc) [inline nix-install.sh:82]:
```suggestion
        printf "\33[1minfo:\33[0m downloading installer \33[4m%s\33[0m\n" "$_url" 1>&2
```

Shouldn't use variables in format srings

**Review by grahamc — COMMENTED** [review]
Looking great.

File: nix-install.sh [hunk 3]
```diff
@@ -241,39 +214,14 @@ get_architecture() {
...
-
-        Android)
-            _ostype=linux-android
-            ;;
-
         Linux)
             check_proc
-            _ostype=unknown-linux-$_clibtype
+            _ostype=linux
             _bitness=$(get_bitness)
```
Comment (grahamc) [inline nix-install.sh:220]: I think we can delete _bitness, get_bitness, get_endianness, is_host_amd64_elf, and a few nearby variables like _clibtype

**Review by grahamc — APPROVED** [review]

**Review by lheckemann — APPROVED** [review]
@Hoverbear is this intentionally still a draft?

**Pull request merged**